
n_channel = 16
overlap = 3.1875
frame_size = 2048
hop = 256
sf = 32000
frame_sample_count = 24000

bag_path = f'/home/pierre-olivier/catkin_ws/src/bag/article/1004-075/'
dict_path = f'/home/pierre-olivier/catkin_ws/src/egonoise/src/database/'

list_bag_noise = ['AL3', 'AL4', 'AL5', 'AL12', 'AL14', 'AL22', 'AL23', 'AL25']
list_bag_database = ['AL1', 'AL2', 'AL11', 'AL15', 'AL21', 'AL24']
bg_intensity = 700

gain = 0.8
list_bag_target = [
['237-126133-0000', 0.8],
['237-126133-0001', 0.8],
['237-126133-0003', 0.8],
['237-126133-0005', 0.8],
['237-126133-0023', 0.8],
['237-126133-0024', 0.8],

['260-123286-0003', 0.8],
['260-123286-0006', 0.8],
['260-123286-0016', 0.8],
['260-123286-0026', 0.8],
['260-123286-0027', 0.8],
['260-123286-0028', 0.8],

['1089-134686-0000', 0.8],
['1089-134686-0002', 0.8],
['1089-134686-0005', 0.8],
['1089-134686-0011', 0.8],
['1089-134686-0012', 0.8],
['1089-134686-0013', 0.8],
#
['1188-133604-0003', 0.8],
['1188-133604-0004', 0.8],
['1188-133604-0007', 0.8],
['1188-133604-0018', 0.8],
['1188-133604-0019', 0.8],
['1188-133604-0021', 0.8],
#
['3570-5694-0000', 0.8],
['3570-5694-0005', 0.8],
['3570-5694-0006', 0.8],
['3570-5694-0007', 0.8],
['3570-5694-0008', 0.8],
['3570-5694-0011', 0.8],
#
['4992-23283-0005', 0.8],
['4992-23283-0009', 0.8],
['4992-23283-0017', 0.8],
['4992-23283-0020', 0.8],
['4992-41797-0008', 0.8],
['4992-41797-0009', 0.8]
]

